main.go

```diff
@@ -115,9 +146,13 @@ func ReadCSVKeywords(path string) (ReaderCSV, error) {
 		if i == 0 || len(line) == 0 {
 			continue
 		}
+		convertInt, err := strconv.ParseInt(line[ColCSVID], 10, 64)
```

Comment:
ты внизу в int32(convertInt) перегоняешь:
```suggestion
		convertInt, err := strconv.ParseInt(line[ColCSVID], 10, 32)
```